<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_plc</class>
 <widget class="QWidget" name="widget_plc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>281</width>
        <height>232</height>
       </rect>
      </property>
      <property name="title">
       <string>变频器</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit_7"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>瓶子直径:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit_6"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>输送线减速时间:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit_5"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>输送线加速时间:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_4"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>理瓶器减速时间:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>秒</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit_3"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>理瓶器加速时间:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Hz</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>理瓶器频率:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>产品速度:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_1"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>BPM</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>毫米</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="radioButton_1">
         <property name="text">
          <string>圆瓶</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>方瓶</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SureButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PlcReturn">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>331</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>光电设置</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>311</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>队列2阈值上限</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>队列2阈值下限</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_12"/>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>队列2测试值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QLineEdit" name="lineEdit_13">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>311</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1" rowspan="2">
         <widget class="QLineEdit" name="lineEdit_16">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>队列3阈值下限</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_14"/>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_15"/>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>队列3测试值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>队列3阈值上限</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>后堵瓶测试值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_20">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>后堵瓶阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_19"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButton_read">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>400</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>22</y>
         <width>311</width>
         <height>48</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>队列1阈值上限</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEdit_9"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>队列1阈值下限</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>队列1测试值</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>311</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>前倒瓶测试值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_18">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>前倒瓶阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_17"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>311</width>
         <height>18</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,1,1">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>报警是否停输送线</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_5">
          <property name="text">
           <string>是</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_6">
          <property name="text">
           <string>否</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4,1,1">
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>报警是否停理瓶器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_7">
          <property name="text">
           <string>是</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_8">
          <property name="text">
           <string>否</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>311</width>
         <height>18</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>报警功能</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_3">
          <property name="text">
           <string>开启</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4">
          <property name="text">
           <string>关闭</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>连续不踢废数量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_21"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>311</width>
         <height>18</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1,1">
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>机型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_9">
          <property name="text">
           <string>左手机</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_10">
          <property name="text">
           <string>右手机</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
